README.md

```diff
@@ -7,3 +7,26 @@ Please leave issues with the website here in this PR thank you
 
 **Where is the chart update / beatmap retrieval scripts?**
 Elsewhere. If any of you twerps want to look at it then I can reveal the algorithm whenever needed
+
+## Developers: setting up with Docker
+
+To set up a development environment, use [Docker compose]. Make sure the Docker
```

Comment:
I wonder whether `sensitiveStrings.php-example` and `sensitiveStrings.php` can be rendered obselete through Docker. Probably worth investigating on my end, I imagine it'll involve being a little more comfortable with Docker before jumping to that...

Reply:
Yes, it's possible and if we were to use Docker compose, then some of the dependent services (db, cache) may not need to be configured entirely since compose will set them up the same way on prod. But that's a bit longer term so I just wanted to put this one here first